projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -1391,23 +1402,80 @@ public function render_outline_label( $id, $label, $required, $required_field_te
 		$classes .= $this->is_error() ? ' form-error' : '';
 		$classes .= $this->label_classes ? ' ' . $this->label_classes : '';
 
+		$output_data = $this->get_outline_styles();
+
 		return '
 			<div class="notched-label">
-				<div class="notched-label__leading"></div>
-				<div class="notched-label__notch">
+				<div class="notched-label__leading ' . $output_data['class_name'] . '"' . $output_data['style'] . '></div>
+				<div class="notched-label__notch ' . $output_data['class_name'] . '"' . $output_data['style'] . '>
 					<label
 						for="' . esc_attr( $id ) . '"
 						class=" ' . $classes . '"
-						style="' . $this->label_styles . '"
+						style="' . $this->label_styles . $output_data['css_vars'] . '"
 					>'
 			. esc_html( $label )
 			. ( $required ? '<span class="grunion-label-required" aria-hidden="true">' . $required_field_text . '</span>' : '' ) .
 			'</label>
 				</div>
-				<div class="notched-label__trailing"></div>
+				<div class="notched-label__trailing ' . $output_data['class_name'] . '"' . $output_data['style'] . '></div>
```

Comment:
Will we need to `esc_attr` each of these?

Reply:
Done!